.github/CODEOWNERS

```diff
@@ -1,27 +1,40 @@
 # Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
 # See https://help.github.com/articles/about-code-owners/
 
-/global.json                    @aspnet/build
-/.azure/                        @aspnet/build
-/.config/                       @aspnet/build
-/build/                         @aspnet/build
-/eng/                           @aspnet/build
-/eng/common/                    @dotnet-maestro-bot
-/eng/Versions.props             @dotnet-maestro-bot @dougbu
-/eng/Version.Details.xml        @dotnet-maestro-bot @dougbu
-/src/Components/                @SteveSandersonMS @dotnet/aspnet-blazor-eng
-/src/DefaultBuilder/            @tratcher
-/src/Hosting/                   @tratcher
-/src/Http/                      @tratcher @jkotalik
-/src/Http/Routing/              @javiercn
-/src/Middleware/                @tratcher
-/src/Middleware/HttpsPolicy/    @jkotalik
-/src/Middleware/Rewrite/        @jkotalik
-# /src/ProjectTemplates/        @ryanbrandenburg
-/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/  @dotnet/aspnet-blazor-eng
+/global.json                                                                      @dotnet/aspnet-build
+/.azure/                                                                          @dotnet/aspnet-build
+/.config/                                                                         @dotnet/aspnet-build
+/.devcontainer/                                                                   @captainsafia
+/.github/                                                                         @dotnet/aspnet-build
+/.github/*_TEMPLATE/                                                              @dotnet/aspnet-build @mkArtakMSFT
+/.github/workflows/                                                               @dotnet/aspnet-build @tratcher
+/docs/                                                                            @captainsafia @mkArtakMSFT
+/eng/                                                                             @dotnet/aspnet-build
+/eng/common/                                                                      @dotnet-maestro-bot
+/eng/Versions.props                                                               @dotnet-maestro-bot @dougbu
+/eng/Version.Details.xml                                                          @dotnet-maestro-bot @dougbu
+/src/Components/                                                                  @dotnet/aspnet-blazor-eng @SteveSandersonMS
```

Comment:
@SteveSandersonMS you don't have this separate assignment in 'release/6.0', 'main' or #35950. Is this needed here❔